core/cmd/local_client_test.go

```diff
@@ -250,11 +250,22 @@ func TestClient_RunNodeWithPasswords(t *testing.T) {
 			set.String("password", test.pwdfile, "")
 			c := cli.NewContext(nil, set, nil)
 
+			run := func() error {
+				cli := cmd.NewApp(&client)
+				if err := cli.Before(c); err != nil {
```

Comment:
Is this call still necessary?

Reply:
Yeah, the config is still constructed there initially